mypy/test/teststubtest.py

```diff
@@ -426,6 +426,16 @@ def test_default_value(self) -> Iterator[Case]:
             error=None,
         )
 
+        # Simulate "<unrepresentable>"
+        yield Case(
+            stub="def f11(text: object = None) -> None: ...",
+            runtime="""
+            def f11(text=None) -> None: pass
+            f11.__text_signature__ = "(text=<unrepresentable>)"
+            """,
+            error=None,
```

Comment:
This isn't the best test, since I think it will pass on master. Maybe change stub to `def f11()` and expect an error?

Reply:
Good point, changed